### PR TITLE
Documentation: Update Hubot Integration documentation

### PR DESCRIPTION
**Pull request by stephenyeargin**
**What is this feature?**

The script package has improved over the years to allow for direct uploading to Slack, etc. This PR updates the documentation to reflect that.

**Why do we need this feature?**

The current documentation says that S3 is required. That is no longer the case.

**Who is this feature for?**

Any user interested in integrating Grafana with Hubot.

**Which issue(s) does this PR fix?**:

None at this time.

**Special notes for your reviewer:**

I would consider this a rather minor improvement.

Please check that:
- [x] It works as expected from a user's perspective.
- [x] If this is a pre-GA feature, it is behind a feature toggle.
- [x] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Comment by Eve832** [comment]
Hey @lwandz13 this is actually for the Hubot data source plugin, can you help push this through? Looks like you'll have to run prettier and attach the required labels

**Comment by lwandz13** [comment]
@stephenyeargin What versions of Grafana does this impact?  Just 10.x?

**Comment by stephenyeargin** [comment]
@lwandz13 The Hubot package will work as far back as v5, accounting for some behavior changes in the v8 API. 😄

File: docs/sources/tutorials/integrate-hubot/index.md [hunk 1]
```diff
@@ -19,16 +19,14 @@ tags:
...
 
 Grafana 2.0 shipped with a great feature that enables it to render any graph or panel to a PNG image.
 
 No matter what data source you are using, the PNG image of the Graph will look the same as it does in your browser.
 
-This guide will show you how to install and configure the [Hubot-Grafana](https://github.com/stephenyeargin/hubot-grafana) plugin. This plugin allows you to tell hubot to render any dashboard or graph right from a channel in Slack, Hipchat or Basecamp. The bot will respond with an image of the graph and a link that will take you to the graph.
-
-> _Amazon S3 Required_: The hubot-grafana script will upload the rendered graphs to Amazon S3. This
-> is so Hipchat and Slack can show them reliably (they require the image to be publicly available).
+This guide will show you how to install and configure the [`hubot-grafana`](https://github.com/stephenyeargin/hubot-grafana) plugin. This plugin allows you to tell hubot to render any dashboard or graph right from a channel in Slack, Basecamp, or any other supported Hubot adapter. The bot will respond with an image of the graph and a link that will take you to the graph.
```
Comment (lwandz13) [inline docs/sources/tutorials/integrate-hubot/index.md:29]:
```suggestion
This guide shows you how to install and configure the [Hubot-Grafana](https://github.com/stephenyeargin/hubot-grafana) plugin. This plugin allows you to tell Hubot to render any dashboard or graph right from a channel in Slack, Basecamp, or any other supported Hubot adapter. The bot will respond with an image of the graph and a link that will take you to the graph.
```

File: docs/sources/tutorials/integrate-hubot/index.md [hunk 2]
```diff
@@ -40,7 +38,7 @@ This guide will show you how to install and configure the [Hubot-Grafana](https:
 
 Hubot is very easy to install and host. If you do not already have a bot up and running please read the official [Getting Started With Hubot](https://hubot.github.com/docs/) guide.
 
-## Install Hubot-Grafana script
+## Install `hubot-grafana` script
```
Comment (lwandz13) [inline docs/sources/tutorials/integrate-hubot/index.md:41]:
```suggestion
## Install the Hubot-Grafana script 
```

File: docs/sources/tutorials/integrate-hubot/index.md [hunk 3]
```diff
@@ -80,9 +75,9 @@ To verify that this feature works try the `Direct link to rendered image` link i
 
 You need to set the environment variable `HUBOT_GRAFANA_API_KEY` to a Grafana API Key. You can add these from the API Keys page which you find in the Organization dropdown.
 
-### Amazon S3
+### Image Uploading
```
Comment (lwandz13) [inline docs/sources/tutorials/integrate-hubot/index.md:78]:
```suggestion
### Image uploading
```

File: docs/sources/tutorials/integrate-hubot/index.md [hunk 4]
```diff
@@ -56,18 +54,15 @@ Edit the file external-scripts.json, and add hubot-grafana to the list of plugin
 
 ## Configure
 
-The `hubot-grafana` plugin requires a number of environment variables to be set in order to work properly.
+The `hubot-grafana` plugin requires two environment variables to be set in order to work properly.
```
Comment (lwandz13) [inline docs/sources/tutorials/integrate-hubot/index.md:57]:
```suggestion
The Hubot-Grafana plugin requires two environment variables to be set in order to work properly. 
```

**Review by lwandz13 — CHANGES_REQUESTED** [review]

**Review by lwandz13 — APPROVED** [review]
Looks good!  Thank you for contributing!

**Pull request merged**